src/Components/CatProfile.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
 import { useState, useEffect, ChangeEvent } from 'react';
-import { getCatsProfiles } from '../api';
+import { getCatsProfiles, deleteCatProfile, updateCatProfile } from '../api';
 import FormInput from './Styling/FormInput';
 import FormElement from './Styling/FormElement';
 import Button from './Styling/Button';
 import { Cat } from '../api';
 
+const loggedInUser = 'USER1';
```

Comment:
Remove this line and change it for something like `const { REACT_APP_LOGGED_IN_USER } = process.env`